libs/fundamental-styles/project.json

```diff
@@ -14,8 +14,14 @@
     "sync": {
       "executor": "@ui5/webcomponents-nx:sync",
       "options": {
-        "configFile": "libs/fundamental-styles/wrapper.conf.ts"
-      }
+        "configFiles": ["libs/fundamental-styles/wrapper.conf.ts", "libs/fundamental-styles/theming.conf.ts"]
+      },
+      "outputs": [
+        "libs/fundamental-styles/components",
```

Comment:
Again, the classes are directives, yet, they are located in `components` folder.